scripts/ci/run-jenkins.sh

```diff
@@ -17,6 +17,7 @@ helix_set_env_vars
 helix_send_build_start_event "build/source/$MONO_HELIX_TYPE/"
 
 make_timeout=300m
+gnumake=$(which gmake || which gnumake || which make)
 
```

Comment:
Shouldn't this be named MAKE ?

Reply:
I thought about it but feared it might conflict with `make`'s internal `$(MAKE)` variable.

Reply:
The two should be the same.